CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid: Samus Returns
 
+- Added: Changing the volume of the music and background ambience is now possible via cosmetic options.
```

Comment:
is background ambience = generic sfx? or is it some other background music thing?

Reply:
Yeah it's the sfx for rooms. Don't have regular sfx yet